src/databases/CGNS/CGNSCommonPluginInfo.C

```diff
@@ -20,7 +38,7 @@
 DatabaseType
 CGNSCommonPluginInfo::GetDatabaseType()
 {
-    return DB_TYPE_MTMD;
+    return dbType;
```

Comment:
Can `GetDatabaseType()` be called before `SetupDatabase()`. Maybe it *can* be but VisIt doesn't wind up operating that way? Reason I ask is that we don't actually know database type until after `SetupDatabase()` so returning a known type (the default) instead of something that indicates that the type is unknown might be a little fib.

Reply:
Actually, I think this is a non-issue. I went looking around for where `GetDatabaseType()` is actually used and I didn't find a single case where that method is called...

```
[scratlantis:~/visit/visit/src] miller86% find . -name databases -prune -false -o \
-name '*.[Ch]' -exec grep 'GetDatabaseType()' {} \; -print
            h << "    virtual DatabaseType              GetDatabaseType();" << endl;
./tools/dev/xml/GenerateInfo.h
    virtual DatabaseType              GetDatabaseType() = 0;
./common/plugin/DatabasePluginInfo.h
`